powerflex-ansible/site.yaml

```diff
@@ -29,44 +29,45 @@
   gather_facts: no
   become: true
   roles:
-   - powerflex-mdm
+   - { role: "powerflex-mdm", when: "'EMC-ScaleIO-mdm' not in ansible_facts.packages" }
   tags: powerflex-mdm
 
 - hosts: tbs
   name: "Install PowerFlex TB"
   gather_facts: no
   become: true
   roles:
-   - powerflex-tb
+   - { role: "powerflex-tb", when: "'EMC-ScaleIO-mdm' not in ansible_facts.packages" }
   tags: powerflex-tb
 
 - hosts: mdms
   name: "Configure PowerFlex storage"
   gather_facts: no
   roles:
-    - powerflex-config
+    - { role: "powerflex-config", when: "'EMC-ScaleIO-mdm' not in ansible_facts.packages" }
   tags: powerflex-config
```

Comment:
MDM of config?

Reply:
The config relies on the mdm cluster presence, if the mdms are there, then the config should be already pushed.